### PR TITLE
♿️ Add alt text to README pnpm logo

### PR DESCRIPTION
**Pull request by k-utsumi**
Selecting the logo image enables us to copy the alt text.

---

**What**:

Improving accessibility and usability.

**Why**:

The links to logo images on GitHub are unnecessary.
Additionally, improve semantics.

**How**:

1. Add h1 heading
2. Remove link to image on GitHub (_ref:_ https://stackoverflow.com/a/48744957/9287284)
3. Set alt text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [Italiano](https://pnpm.io/it/) |
 [Português Brasileiro](https://pnpm.io/pt/)
 
-![](https://i.imgur.com/qlW1eEG.png)
+# <a href="#"> ![pnpm](https://i.imgur.com/qlW1eEG.png) </a>
```
Comment (zkochan) [inline README.md]: why should it be a link?

**Comment by zkochan** [comment]
This looks like a better solution: https://stackoverflow.com/a/73422089

**Comment by k-utsumi** [comment]
Thank you for checking.

After reading [this comment](https://stackoverflow.com/questions/40625614/is-it-possible-to-disable-the-automatic-linking-of-images-in-github-markdown-ren/73422089#comment133355020_73422089), I tried using the picture tag in dark mode, but the image did not appear.
I can't figure out how to solve the problem. . 🫢

Do you want to use the picture tag?

<kbd> image

<details><summary>In Japanese (Before translation)</summary>

ご確認ありがとうございます。

[このコメント](https://stackoverflow.com/questions/40625614/is-it-possible-to-disable-the-automatic-linking-of-images-in-github-markdown-ren/73422089#comment133355020_73422089)を見てダークモード上で picture タグを試してみましたが、確かに画像が表示されませんでした。  
解決方法もパッと分からず。。 🫢

picture タグにしたいですか？

**Comment by zkochan** [comment]
We use the picture tags for the logos of sponsors in the same README file. They seem to work fine in the dark theme.

**Comment by k-utsumi** [comment]
I was mistaken.
When I switched between the `Edit` and `preview` tabs while editing,
the image was displayed even in dark mode.

<kbd> image

File: README.md [hunk 2]
```diff
@@ -4,7 +4,13 @@
...
 [Português Brasileiro](https://pnpm.io/pt/)
 
-![](https://i.imgur.com/qlW1eEG.png)
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://i.imgur.com/qlW1eEG.png">
+    <source media="(prefers-color-scheme: dark)"  srcset="https://i.imgur.com/qlW1eEG.png">
+    <img src="https://i.imgur.com/qlW1eEG.png" alt="pnpm">
+  </picture>
+</h1>
```
Comment (k-utsumi) [inline README.md]: Fixed the image not displaying.

- [Before](https://github.com/pnpm/pnpm/blob/058b540c8f47dabf09ae30a171a808183767f062/README.md) on [♻️ Remove logo link](https://github.com/pnpm/pnpm/pull/8721/commits/058b540c8f47dabf09ae30a171a808183767f062) 058b540c8f47dabf09ae30a171a808183767f062
- ~[After](https://github.com/pnpm/pnpm/blob/8f437f0ab1b4bee19dca3478ca239d625da405d9/README.md) on [🐛 Fix unvisible logo](https://github.com/pnpm/pnpm/pull/8721/commits/8f437f0ab1b4bee19dca3478ca239d625da405d9) 8f437f0ab1b4bee19dca3478ca239d625da405d9~  
  [After](https://github.com/pnpm/pnpm/blob/d0da3fe7368df100a3eec98f47e08fed24e1c4f7/README.md) on [🐛 Fix invisible logo](https://github.com/pnpm/pnpm/pull/8721/commits/d0da3fe7368df100a3eec98f47e08fed24e1c4f7) d0da3fe7368df100a3eec98f47e08fed24e1c4f7

**Comment by k-utsumi** [comment]
I noticed that the`Route4Me` logo wasn't showing, so I'll fix that while I'm at it 🏃‍♂️ 

<kbd> image

<details><summary>Before markdown</summary>

<dl>
<dt> CodeRabbit (visible) </dt>
<dd>

https://github.com/pnpm/pnpm/blob/8f437f0ab1b4bee19dca3478ca239d625da405d9/README.md?plain=1#L108-L114

</dd>
<dt> Route4Me (unvisible) </dt>
<dd>

https://github.com/pnpm/pnpm/blob/8f437f0ab1b4bee19dca3478ca239d625da405d9/README.md?plain=1#L119-L123
</dd>
</dl>

**Comment by zkochan** [comment]
Similar changes should be done in this README: https://github.com/pnpm/pnpm/blob/main/pnpm/README.md

and in the file that generates release notes: https://github.com/pnpm/pnpm/blob/main/__utils__/get-release-text/src/main.ts

**Comment by k-utsumi** [comment]
> Similar changes should be done in this README: https://github.com/pnpm/pnpm/blob/main/pnpm/README.md

I changed. (https://github.com/pnpm/pnpm/pull/8721/commits/ea8e6ada910171e63104d228d241fd6f52c67e90)
Is it okay to ignore the following differences?

<kbd> image

File: README.md [hunk 3]
```diff
@@ -4,7 +4,13 @@
 [Italiano](https://pnpm.io/it/) |
 [Português Brasileiro](https://pnpm.io/pt/)
 
-![](https://i.imgur.com/qlW1eEG.png)
+<h1>
```
Comment (zkochan) [inline README.md]: why is the picture in an h1 tag?

Comment (k-utsumi) [reply README.md]: It follows [markdownlint](https://github.com/DavidAnson/markdownlint#markdownlint).
> [`MD041` - First line in a file should be a top-level heading](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md041.md)

(pnpm has language links, so it does not strictly follow this)

Comment (zkochan) [reply README.md]: There’s no need to add an h1 tag.

Comment (k-utsumi) [reply README.md]: OK, I've removed that ✂️ https://github.com/pnpm/pnpm/pull/8721/commits/2b1a984d438d44ae27377a672d64ef4110d2085f

**Review by zkochan — APPROVED** [review]

**Pull request merged**